<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Slave_interface</class>
 <widget class="QMainWindow" name="Slave_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slave_interface</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/source/background.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="temp_up">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 48pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="temp_down">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>184</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 48pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="preset_temp">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>预设温度</string>
    </property>
   </widget>
   <widget class="QLabel" name="preset_edit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 20pt &quot;华文彩云&quot;;</string>
    </property>
    <property name="text">
     <string>25</string>
    </property>
   </widget>
   <widget class="QLabel" name="curr_temp">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>当前温度</string>
    </property>
   </widget>
   <widget class="QLabel" name="curr_edit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 20pt &quot;华文彩云&quot;;</string>
    </property>
    <property name="text">
     <string>25</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp_adjust">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>温度调节</string>
    </property>
   </widget>
   <widget class="QLabel" name="wind_adjust">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>风速调节</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wind_up">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 48pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="wind_down">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>184</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 48pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="wind">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>风速</string>
    </property>
   </widget>
   <widget class="QLabel" name="cost">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>消费金额</string>
    </property>
   </widget>
   <widget class="QLabel" name="cost_edit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 20pt &quot;华文彩云&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="switch_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/switch_off.jpg);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="operate_mode">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>工作模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="operate_edit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wind_edit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
border-image: url(:/source/mid_wind.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wait_or_work">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>从控机状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="wow_edit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="energy">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>消耗能量</string>
    </property>
   </widget>
   <widget class="QLabel" name="energy_edit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 20pt &quot;华文彩云&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>房间号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="room_edit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>21</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url();
font: 75 20pt &quot;华文彩云&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>temp_adjust</zorder>
   <zorder>temp_up</zorder>
   <zorder>temp_down</zorder>
   <zorder>preset_temp</zorder>
   <zorder>preset_edit</zorder>
   <zorder>curr_temp</zorder>
   <zorder>curr_edit</zorder>
   <zorder>wind_adjust</zorder>
   <zorder>wind_up</zorder>
   <zorder>wind_down</zorder>
   <zorder>wind</zorder>
   <zorder>cost</zorder>
   <zorder>cost_edit</zorder>
   <zorder>switch_2</zorder>
   <zorder>operate_mode</zorder>
   <zorder>operate_edit</zorder>
   <zorder>wind_edit</zorder>
   <zorder>wait_or_work</zorder>
   <zorder>wow_edit</zorder>
   <zorder>energy</zorder>
   <zorder>energy_edit</zorder>
   <zorder>room_label</zorder>
   <zorder>room_edit</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
